sentinelsat/sentinel.py

```diff
@@ -428,6 +428,39 @@ def get_product_odata(self, id, full=False):
         values = _parse_odata_response(response.json()['d'])
         return values
 
+    def _trigger_offline_retrieval(self, url):
+        """ Triggers retrieval of an offline product
+
+        Trying to download an offline product triggers its retrieval from the long term archive.
+        The returned HTTP status code conveys whether this was successful.
+
+        Parameters
+        ----------
+        url : string
+            URL for downloading the product
+
+        Notes
+        -----
+        https://scihub.copernicus.eu/userguide/LongTermArchive
+
+        """
+        # wrapping in closing no longer necessary https://github.com/kennethreitz/requests/pull/4137
```

Comment:
Nice and thanks for the note about `closing`. Please remove the comment though, now that `closing` is gone.

Reply:
Done. This was mostly a note that it could be removed in other places as well.